storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/PostgresStore.scala

```diff
@@ -0,0 +1,123 @@
+package com.twitter.storehaus.postgres
+
+import com.twitter.storehaus.Store
+import com.twitter.util.{Future, Time}
+import roc.postgresql.{Client, Element, Request, Result}
+
+/**
+  * Key value storage with PostgreSQL (>9.5) as backend
+  *
+  * Created by ponkin on 9/24/16.
+  */
+object PostgresStore {
+
+  def apply[K: Show : Read, V: Show : Read](client: Client, table: String, kCol: String, vCol: String) =
+    new PostgresStore[K, V](client, table, kCol, vCol)
+}
+
+class PostgresStore[K: Show : Read, V: Show : Read]
+(protected[postgres] val client: Client, table: String, kCol: String, vCol: String)
+  extends Store[K, V] {
+
+  implicit def toPostgresValue(e: Element): PostgresValue = RocPostgresValue(e)
+
+  private val EMPTY_STRING = ""
+
+  protected val SELECT_SQL_PREFIX =
+    s"SELECT $kCol, $vCol FROM $table WHERE $kCol"
+
+  protected val DELETE_SQL_PREFIX =
+    s"DELETE FROM $table WHERE $kCol"
+
+  override def get(key: K): Future[Option[V]] = {
+    val result = run(selectSQL(List(key)))
```

Comment:
can we pass a limit here? I'm also worried that we may have more than one item. I would rather limit to 2, then do:

```scala
result.flatMap {
  case Nil => Future.None
  case h :: Nil => Future.value(Read[V].read(h.get(Symbol(vCol))))
  case other => Future.exception(new Exception(s"for key: $key expected 0 or 1 items, found: $other")
```